.circleci/do-exclusively.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
This ensures that only one build is running this task at a time. We need this because we use all allowed BrowserStack browsers when running tests against one PR, if we are running tests against two PRs BrowserStack will error saying we are using too many browsers concurrently.